test_sets/persisting_hmi_capabilities.txt

```diff
@@ -22,6 +22,7 @@
 ./test_scripts/Capabilities/PersistingHMICapabilities/020_RAI_use_use_cash_capability.lua
 ./test_scripts/Capabilities/PersistingHMICapabilities/021_Suspending_of_RAI_request_processing_with_another_ccpu_version.lua
 ./test_scripts/Capabilities/PersistingHMICapabilities/022_Suspending_of_RAI_request_processing_in_case_cash_file_does_not_exist.lua
+./test_scripts/Capabilities/PersistingHMICapabilities/022_1_Suspending_of_RAI_request_processing_in_case_cash_file_is_undefined.lua
```

Comment:
@VjKlepikov Please re-enumerate test scripts in a row.

Reply:
Fixed (see [4f166](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/12/commits/4f1669b77acdb41ae2f36e9c2f21e0d6d1ea10ff))